fission-server/src/routes/ipfs.rs

```diff
@@ -0,0 +1,31 @@
+//! IPFS routes
+
+use axum::{
+    http::StatusCode,
+    response::{IntoResponse, Response},
+    Json,
+};
+use serde_json::json;
+
+use crate::{
+    error::{AppError, AppResult},
+    settings::Settings,
+};
+
+use tracing::log;
+
+/// Render a list of IPFS node addresses
+pub async fn peers() -> AppResult<(StatusCode, Response)> {
+    let settings = Settings::load();
```

Comment:
This is going to read the settings file on every request, and will also make testing more challenging by globally tying this piece of state to the filesystem.

It's possible that you might want to read this setting into the `AppState` when the application starts up, which will cut out that filesystem access and also give you a hook for configuring this option during tests.

Reply:
Oh good call! Done 👍